client/src/api/Column.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+// Add column
+const addColumnByBoardId = async (boardId, formData) => {
+  try {
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const body = JSON.stringify(formData);
+
+    const res = await axios.post(
+      `/api/columns/create/${boardId}`,
```

Comment:
similar thing here. Usually we don't need to specify `create`. Just make post request to `api/columns/boardId`